### PR TITLE
Improve attach assertion message

### PR DESCRIPTION
**Pull request by mariusz-jachimowicz-83**
Now the name of problematic key.

### CODE REVIEW

File: src/peripheral/component/attach.clj [hunk 1]
```diff
@@ -51,7 +51,7 @@
   ([component k attach-component]
    (attach component k attach-component nil))
   ([component k attach-component dependencies]
-   (assert (not (contains? component k)) "there is already a component with that key attached.")
+   (assert (not (contains? component k)) (str "there is already a component with " k " key attached."))
```
Comment (xsc) [inline src/peripheral/component/attach.clj]: Grammatically, I think it should be "there is already a component with key XY attached". (Currently it's "[...] with XY key [...]".)

Comment (mariusz-jachimowicz-83) [reply src/peripheral/component/attach.clj]: Right.

**Comment by xsc** [comment]
Thank you!

**Comment by mariusz-jachimowicz-83** [comment]
Cool library. I am very impressed. I am glad to help.

**Comment by xsc** [comment]
Released as:

```clojure
[peripheral "0.5.3"]
```

__[Release Notes](https://github.com/xsc/peripheral/releases/tag/clojars-0.5.3)__

**Pull request merged**